fe/fe-core/src/main/java/org/apache/doris/qe/Coordinator.java

```diff
@@ -471,8 +471,9 @@ public void exec() throws Exception {
                     addressToBackendID.get(execBeAddr),
                     toBrpcHost(execBeAddr),
                     queryOptions.query_timeout * 1000);
-
-            LOG.info("dispatch query job: {} to {}", DebugUtil.printId(queryId), topParams.instanceExecParams.get(0).host);
+            if (LOG.isDebugEnabled()) {
+                LOG.debug("dispatch query job: {} to {}", DebugUtil.printId(queryId), topParams.instanceExecParams.get(0).host);
```

Comment:
Actually this log is useful when tracking a query. Does it really impact the performance?

Reply:
by the flame graph and the statistic provided above，for most users，query performance is the most important thing